std/experimental/typecons.d

```diff
@@ -114,8 +114,12 @@ if (Targets.length >= 1 && allSatisfy!(isMutable, Targets))
                 debug
```

Comment:
There's no need to evaluate the `static if` for an empty block. You could move the version condition e.g. here.
```suggestion
                version (StdUnittest) {}
                else debug
```